packages/devextreme/js/viz/chart.d.ts

```diff
@@ -397,6 +397,325 @@ export interface baseLabelObject {
+};
+
+/**
+ * @docid
+ * @public
+ * @type object
+ * @namespace DevExpress.viz
+ */
+export type StackedPointInfo = BasePointInfo<chartPointObject> & {
+  /**
+   * @docid
+   * @public
+   */
+  percent?: number;
+  /**
+   * @docid
+   * @public
+   */
+  percentText?: string;
+  /**
+   * @docid
+   * @public
+   */
+  points?: StackedPointInfo[];
+  /**
+   * @docid
+   * @public
+   */
+  total?: number;
+  /**
+   * @docid
+   * @public
+   */
+  totalText?: string;
+};
+
+/**
+ * @docid
+ * @public
+ * @type object
+ * @namespace DevExpress.viz
+ */
+export type BubblePointInfo = BasePointInfo<chartPointObject> & {
+  /**
+   * @docid
+   * @public
+   */
+  points?: BubblePointInfo[];
+  /**
+   * @docid
+   * @public
+   */
+  size?: number;
+  /**
+   * @docid
+   * @public
+   */
+  sizeText?: string;
+};
+
+/**
+ * @docid
+ * @public
+ * @type object
+ * @namespace DevExpress.viz
+ */
+export type CandleStickPointInfo = {
+  /**
+   * @docid
+   * @public
+   */
+  argument?: string | number | Date;
+  /**
+   * @docid
+   * @public
+   */
+  closeValue?: number | string;
+  /**
+   * @docid
+   * @public
+   */
+  closeValueText?: string;
+  /**
+   * @docid
+   * @public
+   */
+  highValue?: number | string;
+  /**
+   * @docid
+   * @public
+   */
+  highValueText?: string;
+  /**
+   * @docid
+   * @public
+   */
+  lowValue?: number | string;
+  /**
+   * @docid
+   * @public
+   */
+  lowValueText?: string;
+  /**
+   * @docid
+   * @public
+   */
+  openValue?: number | string;
+  /**
+   * @docid
+   * @public
+   */
+  openValueText?: string;
+  /**
+   * @docid
+   * @public
+   */
+  originalArgument?: string | number | Date;
+  /**
+   * @docid
+   * @public
+   */
+  originalCloseValue?: number | string;
+  /**
+   * @docid
+   * @public
+   */
+  originalHighValue?: number | string;
+  /**
+   * @docid
+   * @public
+   */
+  originalLowValue?: number | string;
+  /**
+   * @docid
+   * @public
+   */
+  originalOpenValue?: number | string;
+  /**
+   * @docid
+   * @public
+   */
+  point?: chartPointObject;
+  /**
+   * @docid
+   * @public
+   */
+  points?: CandleStickPointInfo[];
+  /**
+   * @docid
+   * @public
+   */
+  reductionValue?: number | string;
+  /**
+   * @docid
+   * @public
+   */
+  seriesName?: any;
+  /**
+   * @docid
+   * @public
+   */
+  value?: number | string;
+  /**
+   * @docid
+   * @public
+   */
+  valueText?: string;
+};
+
+/**
+ * @docid
+ * @public
+ * @type object
+ * @namespace DevExpress.viz
+ */
+export type RangePointInfo = {
+  /**
+   * @docid
+   * @public
+   */
+  argument?: string | number | Date;
+  /**
+   * @docid
+   * @public
+   */
+  argumentText?: string;
+  /**
+   * @docid
+   * @public
+   */
+  valueText?: string;
+  /**
+   * @docid
+   * @public
+   */
+  rangeValue1?: string | number | Date;
+  /**
+   * @docid
+   * @public
+   */
+  rangeValue1Text?: string;
+  /**
+   * @docid
+   * @public
+   */
+  rangeValue12?: string | number | Date;
```

Comment:
Shouldn't it be [rangeValue2](https://js.devexpress.com/React/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeAreaSeries/#rangeValue2Field)?

```suggestion
  rangeValue2?: string | number | Date;
```

Reply:
typo, thanks!